apparmor.d/profiles-a-f/flatpak

```diff
@@ -10,6 +10,8 @@ include <tunables/global>
 profile flatpak @{exec_path} flags=(attach_disconnected,mediate_deleted,complain) {
   include <abstractions/base>
   include <abstractions/consoles>
+  include <abstractions/dbus-session-strict>
```

Comment:
Use the dbus abs from this profile (they have to good label): `abstractions/bus-session`